### PR TITLE
translate group label and header type

### PR DESCRIPTION
**Pull request by polarloves**
fix bug:https://github.com/bpmn-io/bpmn-js-properties-panel/issues/537

### CODE REVIEW

File: src/provider/bpmn/BpmnPropertiesProvider.js [hunk 1]
```diff
@@ -206,17 +208,23 @@ function getGroups(element) {
...
+    this._translate = translate;
   }
 
   getGroups(element) {
+    const translate = this._translate;
     return (groups) => {
       groups = groups.concat(getGroups(element));
+      // translate group labels
+      forEach(groups, function(group) {
+        group.label = translate(group.label);
```
Comment (pinussilvestrus) [inline src/provider/bpmn/BpmnPropertiesProvider.js:222]: With this one, we would fix the translation for the `bpmn` properties provider only. We have [multiple ones](https://github.com/bpmn-io/bpmn-js-properties-panel/tree/master/src/provider) for different use cases (Camunda Platform vs. Camunda Cloud specific).

Maybe we should add translation there as well?

**Comment by barmac** [comment]
Instead of monkey-patching the labels, I'd suggest that we translate them explicitly like we already do in the element templates: https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/src/provider/element-templates/ElementTemplatesPropertiesProvider.js#L100

**Review by barmac — CHANGES_REQUESTED** [review]
Cf. https://github.com/bpmn-io/bpmn-js-properties-panel/pull/542#issuecomment-1028718716

**Comment by pinussilvestrus** [comment]
@polarloves any updates on this one?

**Comment by smbea** [comment]
Since there hasn't been any activity on this pull request, I will be closing it for now.

**Pull request closed without merging**